### PR TITLE
Bugfixes `and short form`

### PR DESCRIPTION
**Pull request by IRacle1**

### CODE REVIEW

File: Exiled.API/Extensions/DamageTypeExtensions.cs [hunk 1]
```diff
@@ -23,13 +22,38 @@ public static class DamageTypeExtensions
         /// <summary>
         /// Gets conversion information between <see cref="DeathTranslation.Id"/>s and <see cref="DamageType"/>s.
         /// </summary>
-        public static Dictionary<byte, DamageType> TranslationIdConversion { get; } = // TODO: To readonly
-            TranslationConversion.ToDictionary(key => key.Key.Id, value => value.Value);
+        public static Dictionary<byte, DamageType> TranslationIdConversion { get; } = new() // TODO: ReadOnlyDictionary
```
Comment (IRacle1) [inline Exiled.API/Extensions/DamageTypeExtensions.cs:25]: Remove LINQ, because sometimes it thrown `TranslationConversion` NRE

**Comment by joker-119** [comment]
Was the removal of the separate patch for EnteringFemurBreaker event necessary to fix some sort of bug? 
Because if not, this combining of the two patches shouldn't be done.

**Comment by IRacle1** [comment]
Because `CheckForLurePatch` wasn't working lol, and it's will fix all `Scp106Container` things

**Comment by NaoUnderscore** [comment]
The short form can jump a less amount of bytes; using the short form doesn't mean making everything more maintainable nor optimized, it's just about a logical thing. That said, did you test everything to make sure it works properly without some random weird errors during patch process and/or in-game processes?

**Comment by IRacle1** [comment]
I changed logic only `FailingEscapePocket` and `LureContainer` patch, and yes, i tested

**Comment by joker-119** [comment]
I mean, it's the microest of micro optimizations, but if it's tested and working, I see no issue.

File: Exiled.Events/Patches/Generic/CheckForLurePatch.cs [hunk 2]
```diff
@@ -47,7 +53,12 @@ private static void CheckPlayers()
             {
                 if (Scp106Container.CanBeKilled(player))
                 {
-                    player.Hurt(new UniversalDamageHandler(-1f, DeathTranslations.UsedAs106Bait, null));
+                    var ev = new EnteringFemurBreakerEventArgs(player);
```
Comment (joker-119) [inline Exiled.Events/Patches/Generic/CheckForLurePatch.cs:56]: Please do not use 'var'.
You can use `EnteringFemurBreakerEventArgs ev = new(player);` instead.

**Review by joker-119 — APPROVED** [review]

**Pull request merged**